ui/src/widgets/ui-form/UIForm.vue

```diff
@@ -96,6 +127,8 @@ export default {
                 for (const key in payload) {
                     this.input[key] = payload[key]
                 }
+                // May not be the best way, but if we call this directly will give error on the first `input`, it seems that vue has not yet updated
```

Comment:
Vue has a method for this `this.$nextTick(() => { ... })` which should work here

Reply:
Tanks, i knew it should exist a "better" way, but google was not my friend that day...